src/layer/DigitalGlobeTiledImageLayer.js

```diff
@@ -95,8 +96,12 @@ define([
             this.urlBuilder = {
                 urlForTile: function (tile, imageFormat) {
                     if (!self.metadataRetrievalInProcess) {
-                        return self.urlTemplate.replace("{z}", (tile.level.levelNumber + 1)).
-                            replace("{x}", tile.column).replace("{y}", tile.row);
+                        return self.urlTemplate.replace(
```

Comment:
Webstorm's code reformatting tool was putting all these arguments in a single line that went beyond the recommended width so I manually arranged them in this arguably more readable manner.